src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/PatientTable.java

```diff
@@ -0,0 +1,114 @@
+package com.nha.abdm.wrapper.hrp.mongo.tables;
+
+//import com.nha.abdm.wrapper.hrp.CareContextService;
+import com.nha.abdm.wrapper.hrp.hipInitiatedLinking.responses.LinkRecordsResponse;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.List;
+
+@Document(collection = "patient")
+public class PatientTable {
```

Comment:
Generally the names can just be entities itself without the table suffix, i.e. the name can be  `Patient`, instead of `PatientTable`.